python/script/ci-test

```diff
@@ -5,4 +5,4 @@ set -e
 bundle install
 pyenv exec flake8 helpers/. --count --exclude=./.*,./python/spec/fixtures --show-source --statistics
 bundle exec rubocop .
-bundle exec rspec spec
+bundle exec parallel_test spec/ -n $"CI_NODE_TOTAL" --only-group $"CI_NODE_INDEX" --group-by filesize --type rspec
```

Comment:
There may be typos here, did you mean `$CI_NODE_TOTAL` and `$CI_NODE_INDEX`?

Reply:
No I meant `"$CI_NODE_TOTAL" and `"$CI_NODE_INDEX`

Reply:
Oh, I see, the first quote was off by one :)